scripts/galaxy-main

```diff
@@ -33,7 +33,10 @@ from logging.config import fileConfig
 
 from six.moves.configparser import ConfigParser
 
-from galaxy.util import unicodify
+try:
+    from galaxy.util import unicodify
+except ImportError:
+    unicodify = None
```

Comment:
Can't we simply add `sys.path.insert(1, os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir, 'lib')))` before the import, as we do in most other scripts?

Reply:
We could resolve the symlink if needed, I think.

Reply:
Copied from Pulsar and where it is meant to be pip installable possibly - pip installed stuff should not be mucking with the Python path like that unconditionally? We need to do something like this with all the scripts - but we can do that all at once at some point in the future? I'm indifferent on what is done here for this script.

Reply:
Some day Galaxy will be installable and then we won't need to do any path muckery. Until then...

Reply:
@natefoo I've added a commit to your branch to simplify this, let me know what you think. I haven't had time to add a test for this yet.

Reply:
I've also added an integration test now, which passes for me also under Python 3.